test_scripts/Defects/8_1/3858/3858_2_CreateInteractionChoiceSet_custom_success_codes_error_structure.lua

```diff
@@ -0,0 +1,44 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3858
+---------------------------------------------------------------------------------------------------
+-- Description: Check that SDL transfers succeed result code in error structure from HMI to the mobile app
+-- during processing of CreateInteractionChoiceSet
+
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. Mobile app is registered and activated
+--
+-- Steps:
+-- 1. Mobile app requests CreateInteractionChoiceSet RPC with vrCommands
+-- 2. SDL sends VR.AddCommand(type="Choice") request to the HMI
+-- 3. HMI responds with succeed result code in error structure to VR.AddCommand
```

Comment:
```suggestion
-- 3. HMI responds with success result code in error structure to VR.AddCommand
```

Reply:
All suggested changes have been applied in [542cf75](https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2648/commits/542cf757f8fa71c65866b619aefaa8e22f323935)